src/ui/server/helpers/summary-lists/policy-and-export/single-contract-policy-fields/index.ts

```diff
@@ -34,7 +34,7 @@ const generateSingleContractPolicyFields = (answers: ApplicationPolicyAndExport,
         renderChangeLink: true,
         href: `${INSURANCE_ROOT}/${referenceNumber}${SINGLE_CONTRACT_POLICY_CHANGE}#${CONTRACT_COMPLETION_DATE}-label`,
       },
-      formatDate(answers[CONTRACT_COMPLETION_DATE]),
+      answers[CONTRACT_COMPLETION_DATE] && formatDate(answers[CONTRACT_COMPLETION_DATE]),
```

Comment:
is there a unit test for this?

Reply:
There is - strange because it's got 100% test coverage. I'll add one in anyway